kotlin-checks-test-sources/src/main/kotlin/checks/CommentedCodeCheckSample.kt

```diff
@@ -23,4 +30,22 @@ class CommentedCodeCheckSample {
     /**
      * see [checks.CommentedCodeCheckSample]
      */
+
+    // Compliant, as formatted in backticks
+    // `if (true) println()`
+
+    // Noncompliant@+2
+
+//            val (valid, invalid) = conns.filter { it.isDone }.partition { it.get() == "Deadlock ?" }
+//
+//            println("Completed: ${valid.size} valid, ${invalid.size} invalid of ${conns.size} total [attempts $attempts]")
+
+    // Noncompliant@+2
+
+//    println("This")
+//    println("is")
+//    println("commented-out")
+//    println("code")
+//    println("and")
+//    println("detected")
```

Comment:
Do we have an example of non-code-parsed-as-code in this sample? If so, it would be nice to point it out (as a comment 🙃 )

Reply:
Do we have any example where the code starts immediately after the inline comment tokens like
```kotlin
//callToMethod()
```
It looks like we could detect [such things](https://github.com/SonarSource/slang-test-sources/blob/ff75c5df9b98cd60850e5622970d195a85709a0f/kotlin/kotlin/idea/idea-core/src/org/jetbrains/kotlin/idea/j2k/IdeaDocCommentConverter.kt#L201) in the previous version of the rule.

Reply:
Actually we do have such examples in the ruling it might be related to the expression being short/simple